### ISSUE TITLE
Kakaotalk refusing to show video or voice via a normal call on the app.

### ISSUE THREAD

**Issue by Welved** [tailscale/tailscale#11150]
### What is the issue?

Tailscale seems to be refusing to let a video conferencing app from receiving video and audio. The call goes through to the other party, but... neither one of us is able to view (in the case of a video call) or hear voice (in the case of a voice call).

It had absolutely no issues before, but... it seems as though a admin or someone at tailscale has erected a wall or something.

### Steps to reproduce

Turing on tailscale, doesn't allow the app to function normally. Turning it off lets the app function normally.

### Are there any recent changes that introduced the issue?

Not that I am aware of, I had no problems before.

### OS

Android

### OS version

Android 14/CalyxOS 5.4.0

### Tailscale version

1.58.2

### Other software

Kakaotalk

**Comment by agottardo** [comment]
We ask for a bug report identifier because we cannot diagnose what went wrong without one.

Are you using an exit node when Kakaotalk stops working?

**Comment by Welved** [comment]
Yes, I have my phone setup to use my router as a exit node (gl-mt3000). Had no problems before today.

**Comment by agottardo** [comment]
When you get a chance, please attach a bug report identifier so we can look into it.

**Comment by Welved** [comment]
BUG-2c86c66c32cb0a173cbb08438b1f9dc76410ad4414dc1e6037d715bcd9054059-20240216035734Z-a423b56e0fc74246
